test/unit/conftest.py

```diff
@@ -15,13 +15,15 @@
 
 pytest.register_assert_rewrite('test.unit')
 
+API_VERSIONS = ['v0', 'v1']
```

Comment:
I'm afraid this might get missed when adding v2. Either document it or add autodetection.

Reply:
Will autodetect